onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -3669,8 +3669,7 @@ def aten_ger(self: TensorType, vec2: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-# NOTE: The name is made up for `getitem` to be included in the registry
-@torch_op("aten::getitem")
+@torch_op("_operator::getitem")
```

Comment:
Although it's fixed by #1593, aten::getitem was used in https://github.com/pytorch/pytorch/blob/ee140a198fbe8e7edb5f51806f63e9fa80f523fe/torch/onnx/_internal/fx/onnxfunction_dispatcher.py#L300-L303. This made a regression, and would probably break this week torchbench.

Reply:
Is this part of the refactoring you are doing on dispatcher?

Reply:
hmm ok. We can add the original back

Reply:
Looks like fixed in https://github.com/microsoft/onnxscript/pull/1593